src/nets/e2e_asr_attctc.py

```diff
@@ -570,8 +596,11 @@ def recognize_beam(self, h, recog_args, char_list):
 
         # preprate sos
         y = self.xp.full(1, self.sos, 'i')
-        # maxlen >= 1
-        maxlen = max(1, int(recog_args.maxlenratio * h.shape[0]))
+        if recog_args.maxlenratio == 0:
```

Comment:
please add this specialized behavior in argparse help of asr_recog[_th].py . If you think it is worth being default, set it to 0.0 (now default is 0.5)

Reply:
and I also think argparse should be shared like `from e2e_common import asr_recog_parser` like I said about `def end_detect` because they should be consistent.

Reply:
I'll do it for end_detect, but let me make "argparse part" common later.
Actually, asr_recog.py and asr_recog_th.py are almost same, and I'm thinking of using the common asr_recog.py for both, which is more simple for me, but I'm not sure it would be applicable to asr_train.py and asr_train_th.py, and it needs some more consideration.

Reply:
OK. It may be too much to this PR.